test/httpErrors.test.js

```diff
@@ -40,6 +41,18 @@ test('Should generate the correct http error', t => {
   })
 })
 
+test('Should expose the createError method from http-errors', t => {
+  const fastify = Fastify()
+  fastify.register(Sensible)
+
+  fastify.ready(err => {
+    t.error(err)
+
+    t.is(fastify.httpErrors.createError, createError)
```

Comment:
Could you add a test that throws an error created with that function call?

It would be useful to verify that the custom error handler manage it accordingly, as it would be a surprise for the user if it doesn't work (as I suspect due of this https://github.com/fastify/fastify/issues/2687 )

Reply:
@khendrikse ping

Reply:
O my, sorry for not responding quicker... 

Is this what you had in mind?